client/lcd/lcd_test.go

```diff
@@ -267,21 +267,25 @@ func TestCoinSend(t *testing.T) {
 	require.Equal(t, int64(1), mycoins.Amount.Int64())
 
 	// test failure with too little gas
-	res, body, _ = doSendWithGas(t, port, seed, name, password, addr, 100, 0, "")
+	res, body, _ = doSendWithGas(t, port, seed, name, password, addr, "100", 0, "")
+	require.Equal(t, http.StatusInternalServerError, res.StatusCode, body)
+
+	// test failure with too negative gas
+	res, body, _ = doSendWithGas(t, port, seed, name, password, addr, "-200", 0, "")
 	require.Equal(t, http.StatusInternalServerError, res.StatusCode, body)
```

Comment:
negative gas value should return a client error

Reply:
Not really. The tx runs with gas=AMOUNT, then it eventually fails as expected.